### PR TITLE
Improve readability on active drawer item

### LINKED ISSUE #5350: Dark mode: Hamburger menu has weak focus colors
### Steps to reproduce
1. Set up android to use a dark theme
2. Set up Nextcloud app to follow system
3. Open the app and the hamburger menu (top left button): it brings the left panel into focus, with one item selected.

### Expected behaviour
- The focused item in the left-side menu has vibrant colors

### Actual behaviour
- The focused item in the left-side menu has weak colors: see [screenshot](https://sco.popho.be/s/kB3eRW49NJNo29S) where "_Activities_" is almost unreadable.

### Environment data
Android version: 10

Device model: Nokia 7 Plus

Stock or customized system: Stock

Nextcloud app version: 3.10.0

Nextcloud server version: 18.0.0

#### Web server error log
```
Irrelevant
```

#### Nextcloud log (data/nextcloud.log)
```
Irrelevant
```

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix #5350 

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/main/res/values-night/colors.xml [hunk 1]
```diff
@@ -36,5 +36,6 @@
     <color name="drawer_menu_icon">#ffffff</color>
     <color name="bg_fallback_highlight">#737373</color>
     <color name="switch_track_color_unchecked">#B3FFFFFF</color>
+    <color name="drawer_active_item_background">@color/white</color>
```
Comment (AndyScherzinger) [inline src/main/res/values-night/colors.xml]: fine with me while looking at the non-night color code it probably should be `#80FFFFFF`, no?

Comment (tobiasKaminsky) [reply src/main/res/values-night/colors.xml]: Somewhere there is already a transparency, adding more makes the dark mode less readable…

**Pull request merged**